src/__tests__/react-native-api.test.tsx

```diff
@@ -45,7 +45,7 @@ test('React Native API assumption: nested <Text> renders single host element', (
       </Text>
     </Text>
   );
-  expect(getHostSelf(view.getByText('Hello'))).toBe(view.getByTestId('test'));
+  expect(getHostSelf(view.getByText(/Hello/))).toBe(view.getByTestId('test'));
```

Comment:
is this a breaking change?

Reply:
This test relied of nested Text not being matched, so I needed to change whole string match to partial match. This was a limitation on previous text matching and not a feature per se. IMO that is not a breaking change as test required a minor tweak without affecting the essence of the test.